json/outline.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "Outline",
+    "slug": "outline",
+    "categories": [
+        14
+    ],
+    "date_created": "2024-11-23",
+    "type": "ct",
+    "updateable": true,
+    "privileged": false,
+    "interface_port": "3000",
+    "documentation": "https://docs.getoutline.com/s/guide",
+    "website": "https://www.getoutline.com/",
+    "logo": "https://raw.githubusercontent.com/outline/outline/refs/heads/main/public/images/icon-192.png",
+    "description": "Outline is the fastest knowledge base for growing teams. Beautiful, realtime collaborative, feature packed, and markdown compatible.",
+    "install_methods": [
+        {
+            "type": "default",
+            "script": "ct/outline.sh",
+            "resources": {
+                "cpu": "2",
+                "ram": "4096",
+                "hdd": "8",
+                "os": "debian",
+                "version": "12"
+            }
+        }
+    ],
+    "default_credentials": {
+        "username": null,
```

Comment:
No Default User?

Reply:
No, if you run this script, you will get a screen with "Login to Outline" without any buttons, you need to add an identity provider, since Outline does not manage the users itself. I hope that the notes I wrote in the outline.json are clear enough, or if a link to a discussion should be provided.